libobs/obs-output.c

```diff
@@ -2064,7 +2064,7 @@ static void hook_data_capture(struct obs_output *output)
 		if (has_video)
 			start_raw_video(output->video,
 					obs_output_get_video_conversion(output),
-					default_raw_video_callback, output);
+					0, default_raw_video_callback, output);
```

Comment:
Noting that it uses 0 here as well (also feel free to click resolve button on these if you'd like)

Reply:
also fixed in https://github.com/obsproject/obs-studio/pull/9001/commits/0c1c17056952549c68fcbe9726584fe636bf1e06